app/scripts/controllers/preferences.js

```diff
@@ -403,6 +419,16 @@ export default class PreferencesController {
     return Promise.resolve(tokens);
   }
 
+  async updateTokenType(tokenAddress) {
```

Comment:
We should add some basic documentation for this method, as is done with other token related methods in this file.

Reply:
And maybe a unit test for this method.